src/main/resources/application.properties

```
@@ -18,6 +18,8 @@ rbac/mp-rest/url=https://ci.cloud.redhat.com
 rbac/mp-rest/connectTimeout=2000
 rbac/mp-rest/readTimeout=2000
 
+notifications-backend/mp-rest/url=http://localhost:8085
```

Related to my previous comment:
```suggestion
notifications-backend/mp-rest/url=${clowder.endpoints.notifications-backend-service:http://localhost:8085}
```